.github/workflows/tests.yml

```diff
@@ -0,0 +1,404 @@
+name: GitHub Actions Demo
+run-name: OPEN RUNTIMES TEST
+on: [push]
+jobs:
+  OPEN_RUNTIMES:
+    strategy:
+        fail-fast: false
+        matrix:
+          runtimes: [node-20.4.js, node-20.4.mjs, node-19.0.js, node-19.0.mjs, node-18.0.js, node-18.0.mjs, node-16.0.js, node-16.0.mjs, node-14.5.js, node-14.5.mjs, deno-1.35, deno-1.24, deno-1.21, python-3.11, python-3.10, python-3.9, python-3.8, dart-3.0, dart-2.19, dart-2.18, dart-2.17, dart-2.16, dart-2.15, ruby-3.2, ruby-3.1, ruby-3.0, php-8.2, php-8.1, php-8.0, swift-5.8, swift-5.5, kotlin-1.6, kotlin-1.8, java-18.0, java-17.0, java-11.0, java-8.0, cpp-17, cpp-20, dotnet-7.0, dotnet-6.0]
+          include:
+            - runtimes: node-20.4.js
```

Comment:
Lets add some comments to give some visual separation between the different environments

Please refer to the current travis file